Unable to run v0.26.0

- Your operating system: OS X 10
- Your databases: BigQuery
- Metabase version: 0.26.0
- Metabase hosting environment: Dev Env on Mac
- Metabase internal database: MySQL

Tried running 0.26.0 tag and got following error while starting up:

```
[backend] 09-26 11:58:10 INFO metabase.db :: Verifying mysql Database Connection ...
[backend] 09-26 11:58:11 INFO metabase.db :: Verify Database Connection ...  ✅
[backend] 09-26 11:58:11 INFO metabase.db :: Running Database Migrations...
[backend] Tue Sep 26 11:58:11 PDT 2017 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
[backend] 09-26 11:58:11 INFO metabase.db :: Setting up Liquibase...
[backend] Tue Sep 26 11:58:11 PDT 2017 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
[backend] 
[backend] 09-26 11:58:11 INFO metabase.db :: Liquibase is ready.
[backend] 09-26 11:58:11 INFO metabase.db :: Checking if Database has unrun migrations...
[backend] 09-26 11:58:15 INFO metabase.db :: Database has unrun migrations. Waiting for migration lock to be cleared...
[backend] 09-26 11:58:15 INFO metabase.db :: Migration lock is cleared. Running migrations...
[backend] Exception in thread "main" 
[backend] com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Cannot delete or update a parent row: a foreign key constraint fails, compiling:(/private/var/folders/_f/95xpdnrx4yncxz91dt53mvfh0000gn/T/form-init5407916404786936955.clj:1:125)
[backend] 	at clojure.lang.Compiler.load(Compiler.java:7391)
[backend] 	at clojure.lang.Compiler.loadFile(Compiler.java:7317)
[backend] 	at clojure.main$load_script.invokeStatic(main.clj:275)
[backend] 	at clojure.main$init_opt.invokeStatic(main.clj:277)
[backend] 	at clojure.main$init_opt.invoke(main.clj:277)
[backend] 	at clojure.main$initialize.invokeStatic(main.clj:308)
[backend] 
[backend] 	at clojure.main$null_opt.invokeStatic(main.clj:342)
[backend] 	at clojure.main$null_opt.invoke(main.clj:339)
[backend] 
[backend] 	at clojure.main$main.invokeStatic(main.clj:421)
[backend] 	at clojure.main$main.doInvoke(main.clj:384)
[backend] 
[backend] 	at clojure.lang.RestFn.invoke(RestFn.java:421)
[backend] 	at clojure.lang.Var.invoke(Var.java:383)
[backend] 	at clojure.lang.AFn.applyToHelper(AFn.java:156)
[backend] 	at clojure.lang.Var.applyTo(Var.java:700)
[backend] 	at clojure.main.main(main.java:37)
[backend] Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Cannot delete or update a parent row: a foreign key constraint fails
[backend] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[backend] 
[backend] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[backend] 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[backend] 	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
[backend] 
[backend] 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
[backend] 
[backend] 	at com.mysql.jdbc.Util.getInstance(Util.java:387)
[backend] 
[backend] 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:934)
[backend] 
[backend] 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3966)
[backend] 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3902)
[backend] 	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2526)
[backend] 	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2673)
[backend] 
[backend] 	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2549)
[backend] 
[backend] 	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1861)
[backend] 	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2073)
[backend] 	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2009)
[backend] 	at com.mysql.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:5098)
[backend] 
[backend] 	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1994)
[backend] 
[backend] 	at clojure.java.jdbc$db_do_execute_prepared_statement$fn__4390.invoke(jdbc.clj:901)
[backend] 
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:748)
[backend] 
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
[backend] 
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:697)
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
[backend] 
[backend] 	at clojure.java.jdbc$db_do_execute_prepared_statement.invokeStatic(jdbc.clj:900)
[backend] 	at clojure.java.jdbc$db_do_execute_prepared_statement.invoke(jdbc.clj:894)
[backend] 	at clojure.java.jdbc$db_do_prepared.invokeStatic(jdbc.clj:932)
[backend] 	at clojure.java.jdbc$db_do_prepared.invoke(jdbc.clj:912)
[backend] 	at clojure.java.jdbc$execute_BANG_$execute_helper__4449.invoke(jdbc.clj:1161)
[backend] 	at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:1163)
[backend] 	at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:1145)
[backend] 
[backend] 	at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:1156)
[backend] 	at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:1145)
[backend] 	at metabase.db$migrate_up_if_needed_BANG_.invokeStatic(db.clj:164)
[backend] 
[backend] 	at metabase.db$migrate_up_if_needed_BANG_.invoke(db.clj:151)
[backend] 
[backend] 	at metabase.db$migrate_BANG_$fn__7249.invoke(db.clj:247)
[backend] 
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:714)
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:740)
[backend] 
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
[backend] 
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:697)
[backend] 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
[backend] 	at metabase.db$migrate_BANG_.invokeStatic(db.clj:235)
[backend] 	at metabase.db$migrate_BANG_.invoke(db.clj:218)
[backend] 	at metabase.db$run_schema_migrations_BANG_.invokeStatic(db.clj:365)
[backend] 	at metabase.db$run_schema_migrations_BANG_.invoke(db.clj:360)
[backend] 	at metabase.db$setup_db_BANG_.invokeStatic(db.clj:383)
[backend] 	at metabase.db$setup_db_BANG_.doInvoke(db.clj:376)
[backend] 	at clojure.lang.RestFn.invoke(RestFn.java:421)
[backend] 	at metabase.core$init_BANG_.invokeStatic(core.clj:101)
[backend] 	at metabase.core$init_BANG_.invoke(core.clj:80)
[backend] 	at clojure.lang.Var.invoke(Var.java:375)
[backend] 	at ring.server.standalone$serve.invokeStatic(standalone.clj:95)
[backend] 	at ring.server.standalone$serve.doInvoke(standalone.clj:75)
[backend] 	at clojure.lang.RestFn.invoke(RestFn.java:423)
[backend] 	at ring.server.leiningen$serve.invokeStatic(leiningen.clj:13)
[backend] 	at ring.server.leiningen$serve.invoke(leiningen.clj:10)
[backend] 	at user$eval52505.invokeStatic(form-init5407916404786936
[backend] 955.clj:1)
[backend] 	at user$eval52505.invoke(form-init5407916404786936955.clj:1)
[backend] 	at clojure.lang.Compiler.eval(Compiler.java:6927)
[backend] 	at clojure.lang.Compiler.eval(Compiler.java:6917)
[backend] 	at clojure.lang.Compiler.load(Compiler.java:7379)
[backend] 	... 14 more
[backend] 09-26 11:59:11 INFO metabase.core :: Metabase Shutting Down ...
[backend] 09-26 11:59:11 INFO metabase.core :: Metabase Shutdown COMPLETE
[backend] Subprocess failed
[backend] lein ring server exited with code 1
--> Sending SIGTERM to other processes..
[frontend] yarn run build-hot exited with code 1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/resources/migrations/000_migrations.yaml
+++ b/resources/migrations/000_migrations.yaml
@@ -3836,8 +3836,10 @@ databaseChangeLog:
       id: 66
       author: senior
       comment: 'Added 0.26.0'
+      validCheckSum: '7:e494c2c90fe5c377526da7a6e5ad63a2'
+      validCheckSum: '7:76d06b44a544105c2a613603b8bdf25f'
       changes:
         - sql:
-            sql: drop table if exists computation_job cascade
-        - sql:
             sql: drop table if exists computation_job_result cascade
+        - sql:
+            sql: drop table if exists computation_job cascade

